10_datastructures_lab/02.py

```diff
@@ -1,5 +1,14 @@
-"""
-Write a program that prints only unique values
-from sys.argv (without program name)
-"""
-
+import sys
+file1 = sys.argv[1]
+word_dict = {}
+
+with open(file1, 'r') as fin:
+    for word in fin:
+        word = word.strip('\n')
+        key = sorted(word)
+        key_str = ''.join(key)
+        word_dict.setdefault(key_str, [])
+        if key_str in word_dict:
+            word_dict[key_str].append(word)
+    for key_str in word_dict:
+        print word_dict[key_str]
```

Comment:
👍 
You could eliminate the *key* variable and merger the lines 8-9
Instead of setdefault and lines 9-12 I'd probably write:
```
if key_str in word_dict:
    word_dict[key_str].append(word)
else:
    word_dict[key_str]= [word]
```